pkg/volume/rbd/attacher.go

```diff
@@ -203,7 +223,7 @@ func (detacher *rbdDetacher) UnmountDevice(deviceMountPath string) error {
 		klog.Warningf("Warning: Unmount skipped because path does not exist: %v", deviceMountPath)
 		return nil
 	}
-	devicePath, _, err := mount.GetDeviceNameFromMount(detacher.mounter, deviceMountPath)
+	mp, err := getMountPointFromMounts(detacher.mounter, deviceMountPath)
```

Comment:
Why does `mount.GetDeviceNameFromMount` does not work?

Reply:
`mount.GetDeviceNameFromMount` only return the mounted device path. if we need to mount the device back if detach failed, we need all the mount info, like mount options,  not only the device path. i didn't find the proper function in `mount` package, so this function exists.